apps/scan/backend/src/scanners/custom/app_scan_precinct_config.test.ts

```diff
@@ -89,62 +88,3 @@ test('bmd ballot is accepted if precinct is set for the right precinct', async (
     }
   );
 });
-
-test('hmpb ballot is rejected when scanned for wrong precinct', async () => {
```

Comment:
Isn't this test still important because it tests that the precinct configuration is correctly applied during interp?

Reply:
Isn't that covered by the test above which checks that a BMD ballot of the wrong precinct is rejected, since that confirms precinct state is being passed to interpreter?

Reply:
Oh, didn't see that one. That does make me feel better about it. I still think it's useful to not rely on implementation details for this test (which tests an important property to enforce for HMPB), but I don't feel so strongly to block.